utils/gen-unicode-data/Sources/GenScalarProps/NameAlias.swift

```diff
@@ -0,0 +1,192 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import GenUtils
+
+func getNameAliases(from data: String, into result: inout [(UInt32, String)]) {
+  for line in data.split(separator: "\n") {
+    // Skip comments
+    guard !line.hasPrefix("#") else {
+      continue
+    }
+    
+    let info = line.split(separator: "#")
+    let components = info[0].split(separator: ";")
+    
+    // Name aliases are only found with correction attribute.
+    guard components[2] == "correction" else {
+      continue
+    }
```

Comment:
Could the `alternate` name alias (for ZERO WIDTH NO-BREAK SPACE) also be included?

```
FEFF;BYTE ORDER MARK;alternate
```

And possibly the `control` name aliases? Although how would you choose which one to include?

```
0009;CHARACTER TABULATION;control
0009;HORIZONTAL TABULATION;control
```